CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
   [#518](https://github.com/nextcloud/cookbook/pull/518) @christianlupus
 - Removed dependencies on the global jQuery
   [#497](https://github.com/nextcloud/cookbook/pull/497/) @seyfeb
+- Using computed property in recipe view
+  [#522](https://github.com/nextcloud/cookbook/pull/522/) @seyfeb
```

Comment:
These should be in the unreleased section.

Reply:
oops